python/dgl/transforms/module.py

```diff
@@ -403,6 +404,36 @@ def __call__(self, g):
 
         return g
 
+class DoubleRadiusNodeLabeling(BaseTransform):
```

Comment:
For this item, perhaps it suffices to have the functional interface without a module interface. Module interface helps for transforming multiple graphs in the same way when users provide a transform to a dataset class. However, in this case users need to additionally provide `src` and `dst`. So it's pretty much the same as a functional interface.

Reply:
agree. removed the module interface.